### DISCUSSION TITLE
I want to use global css and module.css.

### DISCUSSION THREAD

**Discussion by Kei-Ta** [vercel/next.js#15513] [Help]
import styles from '../../css/login.module.css'

 <button className="button-basic button-basic-text" {styles.login}>

I have a error.
I want to use global css and module.css.
I want to teach how to use  global css and module.css sametime.

**Comment by jamesmosier** [comment]
If you search around the Discussions and Issues you'll find similar questions/issues as yours. Currently there is no way to do this.

There is an [RFC](https://github.com/vercel/next.js/discussions/15208) that relates to this. 

Or another issue that mentions this as well https://github.com/vercel/next.js/issues/13282
